ql/ABQLRowPluckCore.js

```diff
@@ -0,0 +1,20 @@
+/*
+/*
+ * ABQLRowPluckCore
+ *
+ * An ABQLRowPluckCore can process a value of data and puck out a specified
+ * field to then make an object of values that only contain that field.
+ *
+ */
+
+const ABQLValue = require("./ABQLValue.js");
+const ABQLSetPluck = require("../../platform/ql/ABQLSetPluck.js");
+
+class ABQLRowPluckCore extends ABQLSetPluck {
+}
+
+ABQLRowPluckCore.key = "row_pluck";
+ABQLRowPluckCore.label = "pluck";
+ABQLRowPluckCore.NextQLOps = ABQLValue;
```

Comment:
It looks like here, you are specifying that only ```Value``` level operations can come after a ```RowPluck```, however you'll need to account for transitioning to ```Set``` and ```Row``` level operations depending on what field was selected.